release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3321,6 +3321,10 @@ Contact Red Hat Support for additional details and assistance.
 
 * There is a known issue in this release preventing access to the Web Terminal Operator after it has been installed. This issue will be fixed in a future {product-title} release. (link:https://issues.redhat.com/browse/OCPBUGS-14463[*OCPBUGS-14463*])
 
+* There is a known issue when upgrading the Cluster Network Operator from version 4.13 to 4.14.
```

Comment:
```suggestion
* A known issue exists when you upgrade the Cluster Network Operator from version 4.13 to 4.14.
```
Per IBM Style, avoid starting a sentence with the phrase "There is" or "There are".

Reply:
I was following the pattern here:
https://redhat-documentation.github.io/supplementary-style-guide/#_known_issue
Does the supplementary guide take precedence over IBM or the other way around?